src/controller/ConferenceCtrl.java

```diff
@@ -0,0 +1,55 @@
+package controller;
+
+import javax.validation.Valid;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import domain.Conference;
+import domain.User;
+import repo.ConferenceRepository;
+import service.UserService;
+import util.BaseController;
+
+@Controller
+public class ConferenceCtrl extends BaseController
+{
+
+    @RequestMapping(value = "/conference", method = RequestMethod.GET)
```

Comment:
Redenumeste /conference in /createConference sa stim explicit despre ce face actiunea.